src/openrct2-ui/interface/Window.cpp

```diff
@@ -212,14 +212,16 @@ rct_window* window_create_auto_pos(
     {
         if (w->flags & WF_STICK_TO_BACK)
             continue;
-        const std::array offsets = { ScreenCoordsXY{ w->width + 2, 0 },
-                                     ScreenCoordsXY{ -w->width - 2, 0 },
-                                     ScreenCoordsXY{ 0, w->height + 2 },
-                                     ScreenCoordsXY{ 0, -w->height - 2 },
-                                     ScreenCoordsXY{ w->width + 2, -w->height - 2 },
-                                     ScreenCoordsXY{ -w->width - 2, -w->height - 2 },
-                                     ScreenCoordsXY{ w->width + 2, w->height + 2 },
-                                     ScreenCoordsXY{ -w->width - 2, w->height + 2 } };
+
+        const ScreenCoordsXY offsets[] = { { w->width + 2, 0 },
```

Comment:
`std::array` requires each of the elements to be explicitly constructed with ScreenCoordx{}, whereas a regular array does not require this. IMO this makes for more readable code.

If this is not desirable, we can drop the commit.

Reply:
Splitting this patch off to another PR. Will open after this PR has been merged.